serverless/src/submitHarmonyOrder/ccwShapefile.js

```diff
@@ -0,0 +1,36 @@
+import { isClockwiseLatLng } from '../../../static/src/js/util/map/granules'
+
+/**
```

Comment:
Not sure we need to do it now, but this seems like it'd be useful as a shared util.